sentry-android-core/src/test/java/io/sentry/android/core/InternalSentrySdkTest.kt

```diff
@@ -199,20 +199,16 @@ class InternalSentrySdkTest {
 
     @Test
     fun `current scope returns obj when scopes is active`() {
```

Comment:
Are these tests now testing something different? Should they be additions instead of replacements?

Reply:
With `Sentry.close()` in test setup, `InternalSentrySdk.getCurrentScope()` doesn't work correctly afair, so I used the actual setup code. Tests should still test the same thing now.

I just retested with the old code and it seems to work so maybe this  was just an interim thing. I do however think the new code should be closer to real world usage, so I'd keep it this way.

wdyt?